source/index.blade.php

```diff
@@ -70,7 +70,7 @@
               </ul>
             </div>
             <div class="ud-hero-image wow fadeInUp" data-wow-delay=".25s">
-              <img src="assets/images/hero/hero-image.svg" alt="hero-image" />
+              <img src="assets/images/hero/print_librecode.png" alt="hero-image" />
```

Comment:
What is hero at this context?

Maybe would be good a new PR  replacing by another name. What you think about this?

But maybe is an unnecessary change.

Reply:
Was created a new issue